src/generator.js

```diff
@@ -23,15 +22,18 @@ function generator(fn, localPropNames, options) {
     };
     var globalizePropNames = commonPropNames.concat(localPropNames);
 
-    return reactCreateClass({
-        displayName: Fn,
-        componentWillMount: function() {
+    return class extends React.Component {
+        // static displayName = Fn;
```

Comment:
Uncomment?

Reply:
This is the *Outstanding issues* from the description. Esperanto chokes when this is hit, even with the babel plugin.

Reply:
Ah, I see

Reply:
Does that mean we need the tool improvements too?

Reply:
They will be needed if you want to support `displayName`. But that's really only for debugging purposes, so it's not a blocker for things to work.